Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -61,33 +60,24 @@
 
 upgradeProperties[2017] = {
     '2017' : {
-        'Geom' : 'DB:Extended',
+        'Geom' : 'Extended2017new',
         'GT' : 'auto:phase1_2017_realistic',
         'Era' : 'Run2_2017',
         'ScenToRun' : ['GenSimFull','DigiFull','RecoFull','ALCAFull','HARVESTFull'],
     },
-    '2017NewFPix' : {
-        'Geom' : 'Extended2017NewFPix',
-        'GT' : 'auto:phase1_2017_realistic',
-        'Era' : 'Run2_2017_NewFPix',
-        'ScenToRun' : ['GenSimFull','DigiFull','RecoFull','ALCAFull','HARVESTFull'],
-    },
-    '2017HCALdev' : {
-        'Geom' : 'Extended2017dev',
-        'GT' : 'auto:phase1_2017_hcaldev',
-        'Era' : 'Run2_2017_HCALdev',
-        'ScenToRun' : ['GenSimFull','DigiFull','RecoFull','ALCAFull','HARVESTFull'],
-    },
-    '2017AllNew' : {
+    '2017Design' : {
         'Geom' : 'Extended2017new',
-        'GT' : 'auto:phase1_2017_hcaldev',
-        'Era' : 'Run2_2017_new',
-        'ScenToRun' : ['GenSimFull','DigiFull','RecoFull','ALCAFull','HARVESTFull'],
+        'GT' : 'auto:phase1_2017_design',
+        'Era' : 'Run2_2017',
+        'BeamSpot': 'GaussSigmaZ4cm',
+        'ScenToRun' : ['GenSimFull','DigiFull','RecoFull','HARVESTFull'],
```

Comment:
since ALCAFull is mostly for alignment purposes and since the underlying geometry is the same as the regular workflows, while different alignments can be applied for studies via track refit, I tend to second @makortel's opinion